src/apps/common/submodules/navigationWizard/navigationWizard.scss

```diff
@@ -109,19 +112,10 @@
 }
 
 #navigation_wizard_wrapper #cancel {
-	float: right;
-	bottom: 38px;
-	right: 25px;
 	min-width: 72px;
 	width: 72px;
 }
 
-#navigation_wizard_wrapper #cancel, #navigation_wizard_wrapper .back {
```

Comment:
by removing this now the cancel button and back are just plain gray, I think you should be this section

Reply:
you are the best, thanks!